packages/elements/src/internals/machines/sign-in/router.machine.ts

```diff
@@ -204,7 +204,7 @@ export const SignInRouterMachine = setup({
         },
       })),
     },
-    'AUTHENTICATE.SAML': {
+    'AUTHENTICATE.ENTERPRISE_SSO': {
```

Comment:
@LauraBeatris Any reason that we're updating `SAML` to `ENTERPRISE_SSO` on sign-in while simply appending `ENTERPRISE_SSO` on sign-up?

Reply:
Oh, this is wrong - we should keep supporting both, let me fix it here, good catch!

Reply:
Fixed on https://github.com/clerk/javascript/pull/4916/commits/941a14a8e474422f24f5b874456b43fc9739cfef